index.ts

```diff
@@ -121,11 +120,11 @@ export class Server {
             }
 
             res.status(500).json(errorBody);
-        });
+        }));
     }
 
     protected configureCatchAll() {
-        this._express.use((req, res) => {
+        this._express.use((_req, res) => {
```

Comment:
Why the underscore?

Reply:
It's a convention to mark unused positional args with underscores. Stops the IDE/typescript from complaining about them.